woocommerce/woocommerce
# Fix attribute list styles

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PR fixes a few style errors in the attribute list in the blocks product editor.

Before

<img width="789" alt="Screenshot 2023-07-13 at 11 11 30 AM" src="https://github.com/woocommerce/woocommerce/assets/2240960/740dd8d3-7891-4cd8-aca0-62a6a7ce6052">

After

![Screenshot 2023-09-22 at 14 31 09](https://github.com/woocommerce/woocommerce/assets/1314156/dbdb64db-53df-418c-900e-aec2019a3f52)

Closes #39224.


**Acceptance criteria**

- [ ] List: the color of the delete button should be Gutenberg-700, not Gutenberg-900 ( $gray-700 ).
- [ ] List: the right padding on the list item should be 0px, not 24px. 




### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Enable the product blocks editor under WooCommerce -> Settings -> Advanced -> Features.
2. Make sure feature `product-variation-management` is enabled under `Features` tab from `/wp-admin/tools.php?page=woocommerce-admin-test-helper`.
3. Navigate to the Add Product page.
4. Go to `Variations` and add a variation.
5. Check the acceptance criteria. 



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
